_pages/user-experience/failure-proof.md

```diff
@@ -13,12 +13,49 @@ sidenav:
     href: "user-experience/cancel-url/"
   - text: Failure to proof URL
     href: "user-experience/failure-proof/"
+    links:
+      - text: What is a Failure to Proof URL?
+        href: "user-experience/failure-proof/#what-is-a-failure-to-proof-url"
+      - text: Where should the Failure to Proof URL take users?
+        href: "user-experience/failure-proof/#where-should-the-failure-to-proof-url-take-users"
   - text: Knowledge articles
     href: "user-experience/knowledge-articles/"
-  - text: FAQ content 
+  - text: FAQ content
     href: "user-experience/faq-content/"
 
 ---
 
-## Failure Proof
+## Identity Verification: Create a Failure to Proof URL
+Applications with Login.gov identity verification need to provide an alternative proofing path to support users who are unable to complete Login.gov’s process.
 
+### What is a Failure to Proof URL?
+Login.gov’s agency partners provide a link that their users are directed to if they are unable to verify their identity with Login.gov. This is called the Failure to Proof URL (e.g. `failure_to_proof_url`). This link routes users to a designated page on the agency partner’s site which is configured as part of the agency's integration.
+
+The Failure to Proof URL is shown to users throughout the identity proofing process. Some examples of when a user might encounter this are:
+
+- Users might navigate to this page if they don’t have a valid state-issued ID or Social Security number.
+- The Failure to Proof URL is available for users who are not able  to capture or upload photos of their ID.
+- Users see the Failure to Proof URL when they encounter errors verifying their ID, personal information, or phone number. Examples of this include:
+- Users who enter valid information, but are not able to pass Login.gov’s proofing process because we cannot verify their information.
```

Comment:
Should this bullet and the next be indented? The bullet above ends with a colon.

Reply:
ooh yes thanks

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/414/commits/8a69618f0a0501dfe6bff39fd501bf240cadda5c